specs/source/management/ext-jwt-signers.yml

```diff
@@ -0,0 +1,242 @@
+        '200':
+          $ref: '../shared/standard-responses.yml#/responses/patchResponse'
+        '400':
+          $ref: '../shared/standard-responses.yml#/responses/badRequestResponse'
+        '404':
+          $ref: '../shared/standard-responses.yml#/responses/notFoundResponse'
+        '401':
+          $ref: '../shared/standard-responses.yml#/responses/unauthorizedResponse'
+    delete:
+      summary: Delete an External JWT Signer
+      description: |
+        Delete an External JWT Signer by id. Requires admin access.
+      security:
+        - ztSession: [ ]
+      tags:
+        - External JWT Signer
+      operationId: deleteExternalJwtSigner
+      responses:
+        '200':
+          $ref: '../shared/standard-responses.yml#/responses/deleteResponse'
+        '400':
+          $ref: '../shared/standard-responses.yml#/responses/badRequestResponse'
+        '401':
+          $ref: '../shared/standard-responses.yml#/responses/unauthorizedResponse'
+responses:
+  listExternalJwtSigners:
+    description: A list of External JWT Signers
+    schema:
+      $ref: '#/definitions/listExternalJwtSignersEnvelope'
+  detailExternalJwtSigner:
+    description: A singular External JWT Signer resource
+    schema:
+      $ref: '#/definitions/detailExternalJwtSignerEnvelope'
+
+definitions:
+  listExternalJwtSignersEnvelope:
+    type: object
+    required:
+      - meta
+      - data
+    properties:
+      meta:
+        $ref: '../shared/standard-responses.yml#/definitions/meta'
+      data:
+        $ref: '#/definitions/externalJwtSignerList'
+  detailExternalJwtSignerEnvelope:
+    type: object
+    required:
+      - meta
+      - data
+    properties:
+      meta:
+        $ref: '../shared/standard-responses.yml#/definitions/meta'
+      data:
+        $ref: '#/definitions/externalJwtSignerDetail'
+  externalJwtSignerList:
+    description: An array of External JWT Signers resources
+    type: array
+    items:
+      $ref: '#/definitions/externalJwtSignerDetail'
+  externalJwtSignerDetail:
+    description: A External JWT Signer resource
+    type: object
+    allOf:
+      - $ref: '../shared/base-entity.yml#/definitions/baseEntity'
+      - type: object
+        required:
+          - name
+          - certPem
+          - enabled
+          - fingerprint
+          - commonName
+          - notAfter
+          - notBefore
+        properties:
+          name:
+            type: string
+            example: 'MyApps Signer'
+          certPem:
+            type: string
+          enabled:
+            type: boolean
+          fingerprint:
+            type: string
+          commonName:
+            type: string
+          notAfter:
+            type: string
+            format: date-time
+          notBefore:
+            type: string
+            format: date-time
+  externalJwtSignerCreate:
+    description: A create Certificate Authority (CA) object
+    type: object
+    required:
+      - name
+      - certPem
+      - enabled
```

Comment:
Are commonName, fingerprint, notBefore and notAfter all set automatically by decoding the cert?

Reply:
Yes. They are there for informational purposes on read only.